### DISCUSSION TITLE
Welcome to E3SM Discussions!

### DISCUSSION THREAD

**Discussion by rljacob** [E3SM-Project/E3SM#4418] [Announcements]
## 👋 Welcome!
  We’re testing github Discussions as a place to connect with other E3SM users and developers. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas for new features.
  * Engage with other community members.
  * Welcome others and are open-minded. 

The content is world-readable.  A free github account is needed to post new items or make comments.

Continue to use https://github.com/E3SM-Project/E3SM/issues for reporting bugs.  Use this for asking questions, making requests and other discussion.

**Comment by TonyB9000** [comment]
### Discussion Structure:
It would be great if discussions had a "titlebar", and an automatic of table of contents of "titles" allowed jumping/branching to specific discussions, sort-able in various ways.  I wonder what this will look like once hundreds of discussion are represented. (Will I ever give up spamming people with my emails...?  They are easily organized into folders and sort-able by date, author, content...)

**Reply by rljacob** [reply]
These are the default categories.  We can make more:
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository

Here's an repo with a lot of items in their Discussion:  https://github.com/tailwindlabs/tailwindcss/discussions

**Reply by TonyB9000** [reply]
Excellent resource.  Thanks Rob.
